mxnet-bot/LabelBotFullFunctionality/LabelBot.py

```diff
@@ -303,6 +358,11 @@ def parse_webhook_data(self, event):
                     logging.error(f'Unsupported actions: {actions}')
                     raise Exception("Unrecognized/Infeasible label action for the mxnet-label-bot")
 
+        # On creation of a new issue, automatically trigger the bot to recommend labels
+        if github_event == "issues" and payload["action"] == "opened":
+            self._find_all_labels()
```

Comment:
Why do you call _find_all_labels here? Doesn't seem to be used in the call to predict_label below

Reply:
predict_label makes a call to add_label and add_label requires that we find all labels first (i.e. the assert statement initially in add_label)

Reply:
Or maybe even have it in add_label if it is that is the function that requires it? Your call though

Reply:
Following the previous convention of find_all_labels in parse_webhook_data. I think its best we stick with making the call here